docs/INSTALL.md

```diff
@@ -0,0 +1,83 @@
+## Prerequisite
+If you are using kubernetes distribution with no running OLM you can use [operator-sdk] cli
+```shell
+$ operator-sdk olm install
+```
+
+## Installation
+To get a functioning remediation mechanism you can go to the operator page on [operator hub]
+
+Alternatively you can run`$ kubectl create -f https://operatorhub.io/install/node-healthcheck-operator.yaml`
+
+By default, OLM will resolve Poison-Pill as a dependency and will install the latest available version
+in the current catalog or other catalog with higher priority.
+
+## Post install
+
+>NOTE: OLM operators goes under `operators` namespace by default.
+>      The Poison pill config and other resource in this example goes under `default` but 
+>      can go under every desired namespace.
+
+- create `PoisonPillConfig` object ([ongoing work to make it automatic][ppil-auto-config])
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: poison-pill.medik8s.io/v1alpha1
+kind: PoisonPillConfig
+namespace: default
+metadata:
+  name: poison-pill-config
+spec: {}
+EOF
+```
+- create a poison pill remediation template
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: poison-pill.medik8s.io/v1alpha1
+kind: PoisonPillRemediationTemplate
+namespace: default
+metadata:
+  name: ppill-template
+spec: {}
+```
+
+- create Node-Healthcheck CR that points to poison-pill remediation
+```shell
+cat << EOF | kubectl create -f -
+apiVersion: remediation.medik8s.io/v1alpha1
+kind: NodeHealthCheck
+namespace: default
+metadata:
+  name: nodehealthcheck-sample
+spec:
+  unhealthyConditions:
```

Comment:
I think we should also add Ready-False check here, just like MHC

Reply:
ack